views/index.hbs

```diff
@@ -1,2 +1,10 @@
 <h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
\ No newline at end of file
+<p>Welcome to {{title}}</p>
+
+<a href="/celebrities/create/">Create Celebrity</a>
+<a href="/celebrities/">List of Celebrities</a>
+<br>
+<a href="movies/create"> Create New Movie</a>
```

Comment:
You're missing the "/" on the `href` here. 
`<a href="/movies/create"> Create New Movie</a>`